lib/client.js

```diff
@@ -0,0 +1,269 @@
+/*!
+ * clients.js - client extensions of bclient to connect to bpanel app server
+ * Copyright (c) 2018, Bcoin Devs (MIT License).
+ * https://github.com/bcoin-org/bpanel-utils
+ */
+
+'use strict';
+
+import bclient from 'bclient';
+import hsclient from 'hs-client';
+import { Client } from 'bcurl';
+import assert from 'bsert';
+import MultisigClient from 'bmultisig/lib/client';
+
+const { NodeClient: BNodeClient, WalletClient: BWalletClient } = bclient;
+const { NodeClient: HNodeClient, WalletClient: HWalletClient } = hsclient;
+
+class BPClient extends Client {
+  /**
+   * Create a client for use in bPanel.
+   * @constructor
+   * @param {Object} options
```

Comment:
Maybe add some more information about what the options are?

I can see from this line: `const { id, chain, path, node, wallet, multisig } = options;` what they are
but its digging a bit into the code

Reply:
done